team/end-of-iteration-demos.md

```diff
@@ -0,0 +1,12 @@
+---
+title: End of iteration demos
+description: Time to celebrate the great work completed at the end of the iteration
+---
+
+Our team's demo session is an hour on the last day of the iteration. This is a chance to share and celebrate the amazing work done during the iteration. The purpose of this meeting is to give some time to look back at the work completed and to share it with the rest of the team. It's key that everyone is aware of what each other are working on so we don't design and develop in silos.
+
+## Demo completed work
```

Comment:
I'm not crazy about the idea of enforcing this. I think we should always encourage people to share their work, and demo anything they're proud of.

I don't really see any downsides to allowing people to demo anything, but if we do get issues with, for example, people demoing the same work over and over again, I'd rather we address it after we have the problem rather than trying to preempt the problem.

Reply:
Well this was the main point of this document and so it seems to agree on this point as a team before this can be committed too.